pkg/k8s/k8s.go

```diff
@@ -193,3 +194,30 @@ func KindToL5DLabel(k8sKind string) string {
 	}
 	return k8sKind
 }
+
+// PodIdentity returns the mesh TLS identity name of this pod, as constructed
+// from the pod's service account name and other metadata.
+func PodIdentity(pod *corev1.Pod) (string, error) {
+	if pod.Status.Phase != corev1.PodRunning {
+		return "", fmt.Errorf("pod not running: %s", pod.GetName())
+	}
+
+	podsa := pod.Spec.ServiceAccountName
+	podns := pod.ObjectMeta.Namespace
+	for _, c := range pod.Spec.Containers {
+		if c.Name == ProxyContainerName {
+			var l5dns string
+			var l5dtrustdomain string
```

Comment:
TIOLI: could also be written as:
```suggestion
			var l5dns, l5dtrustdomain string
```